src/main/java/net/imagej/plugins/uploaders/webdav/ProgressHttpEntityWrapper.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * The MIT License (MIT)
```

Comment:
This file should have our usual license on top, and then the original code's MIT license beneath it. Annoying I know, but that's third party attribution for you...